aws_promise/lambda.js

```diff
@@ -53,5 +53,21 @@ module.exports = {
       params['SourceArn'] = input.sourceArn;
     }
     return lambda.addPermission(params).promise();
+  },
+
+  invokeFunction: function(input) {
+    var lambda = this.findService(input);
+    var invocationType= (input.invocationType) ? input.invocationType : "RequestResponse";
+    var logType = (input.logType) ? input.logType : "Tail";
+    var qualifier= (input.qualifier) ? input.qualifier : "$LATEST";
```

Comment:
add  below line to make payload string from object and pass this string in payload
    var payload = JSON.stringify(input.payload);

Reply:
Done.